release-notes/opensearch-security-dashboards-plugin.release-notes-2.14.0.0.md

```diff
@@ -0,0 +1,21 @@
+## Version 2.14.0.0
+
+Compatible with OpenSearch-Dashboards 2.14.0
+
+### Enhancements
+* Adds Multiple Datasources Support for Security Dashboards Plugin ([#1888](https://github.com/opensearch-project/security-dashboards-plugin/pull/1888)) 
+* Adds flow-framework transport action permissions to the static dropdown list ([#1908](https://github.com/opensearch-project/security-dashboards-plugin/pull/1908)) 
+* Update copy for tenancy tab ([#1881](https://github.com/opensearch-project/security-dashboards-plugin/pull/1881))
+* Clear session storage on auto-logout & remove unused saml function ([#1872](https://github.com/opensearch-project/security-dashboards-plugin/pull/1872))
+* Create a password strength UI for security dashboards plugin ([#1762](https://github.com/opensearch-project/security-dashboards-plugin/pull/1762))
+
+### Bug Fixes
+* Fixes saml login flow to work with anonymous auth ([#1839](https://github.com/opensearch-project/security-dashboards-plugin/pull/1839))
+* Fixes issue with multi-tenancy and default route to use corresponding default route for the selected tenant ([#1820](https://github.com/opensearch-project/security-dashboards-plugin/pull/1820))
+* Fix oidc cypress test and remove doc link ([#1923](https://github.com/opensearch-project/security-dashboards-plugin/pull/1923))
```

Comment:
minor nit: I don't think this was backported to 2.14. It is a test fix, and I just checked now - OSD merged in the PR that prompted this PR in last night. @cwperks do you think we should also backport this to 2.14 in case we need to add anything to the release branch? https://github.com/opensearch-project/OpenSearch-Dashboards/commit/521b943049044bc380fbd03f16ccd0487d4d4384

Reply:
yes let's backport it

Reply:
Resolving this comment since the backport PR was raised: https://github.com/opensearch-project/security-dashboards-plugin/pull/1929. Once that is merged, release notes look good to me.